src/pyrokinetics/normalisation.py

```diff
@@ -364,6 +384,73 @@ def default_convention(self, convention):
         self._current_convention = self._conventions[convention]
         self._update_references()
 
+    def add_convention_normalisation(self, name=None, convention_dict=None):
```

Comment:
If you want to go even further, could add a `TypedDict` to explicitly document what keys `convention_dict` should have. Possibly more effort than it's worth though!